commit b40dff9873fa9393484bdd92006577fe5b2e0fc0
Author: Michael Still <mikal@stillhq.com>
Date:   Mon Sep 14 18:32:13 2026 +1000

    Convert the storage services to JSON logging.
    
    Cinder and Manila now honour openstack_logging_json.
    
    Change-Id: I8d9036bc819ec0b45d2dc1b489138fb07890c8be
    Signed-off-by: Michael Still <mikal@stillhq.com>

diff --git a/ansible/roles/cinder/templates/cinder.conf.j2 b/ansible/roles/cinder/templates/cinder.conf.j2
index fc0dd2dfa..a2c638297 100644
--- a/ansible/roles/cinder/templates/cinder.conf.j2
+++ b/ansible/roles/cinder/templates/cinder.conf.j2
@@ -1,5 +1,6 @@
 [DEFAULT]
 debug = {{ cinder_logging_debug }}
+use_json = {{ openstack_logging_json }}
 
 log_dir = /var/log/kolla/cinder
 {% if service_name == "cinder-api" %}
diff --git a/ansible/roles/manila/templates/manila.conf.j2 b/ansible/roles/manila/templates/manila.conf.j2
index 148989d9a..7e3eb1a3c 100644
--- a/ansible/roles/manila/templates/manila.conf.j2
+++ b/ansible/roles/manila/templates/manila.conf.j2
@@ -1,5 +1,6 @@
 [DEFAULT]
 debug = {{ manila_logging_debug }}
+use_json = {{ openstack_logging_json }}
 
 log_dir = /var/log/kolla/manila
 {% if service_name == "manila-api" %}
